run_trainer.sh

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env bash
```

Comment:
this needs to be fixed. This is a GPU script used to run TRN. We need to use the TRN script and just add ckpt resume to it.